docs/components/connectors/use-connectors/index.md

```diff
@@ -14,6 +14,10 @@ New to modeling with Camunda? The steps below assume some experience with Camund
 
 ## Using secrets
 
+:::warning
+`secrets.*` is a deprecated syntax. Please use instead `{{secrets.*}}`
```

Comment:
```suggestion
`secrets.*` is a deprecated syntax. Please use `{{secrets.*}}` instead.
```

Reply:
In this case, I would actually just say "Instead, use `{{secrets.*}}`."